genkit-tools/common/src/runner/runner.ts

```diff
@@ -258,7 +269,16 @@ export class Runner {
           this.appProcess = null;
           resolve();
         });
-        this.appProcess.kill();
+        if (process.platform === 'win32') {
```

Comment:
have you looked at https://www.npmjs.com/package/terminate?

Reply:
No, good suggestion, will try this!